script.service.hue/resources/lib/kodilogging.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+import logging
+from logging.handlers import TimedRotatingFileHandler
+import os
+
+import xbmc
+
+from . import ADDONID, ADDON, ADDONDIR
+
+prefix = "[{}]".format(ADDONID)
+formatter = logging.Formatter(prefix + '[%(module)s][%(funcName)s](%(lineno)d): %(message)s')
+fileFormatter = logging.Formatter('%(asctime)s %(levelname)s [%(module)s][%(funcName)s](%(lineno)d): %(message)s')
+
+
+class KodiLogHandler(logging.StreamHandler):
+
+    def __init__(self):
+        logging.StreamHandler.__init__(self)
+        self.setFormatter(formatter)
+
+    def emit(self, record):
+        levels = {
+            logging.CRITICAL: xbmc.LOGFATAL,
+            logging.ERROR: xbmc.LOGERROR,
+            logging.WARNING: xbmc.LOGWARNING,
+            logging.INFO: xbmc.LOGINFO,
+            logging.DEBUG: xbmc.LOGDEBUG,
+            logging.NOTSET: xbmc.LOGNONE,
+        }

```

Comment:
please only use the xbmc.LOGDEBUG level.
we really would like to keep the regular kodi.log file as clean (and small) as possible.

Reply:
I will change it throughout. I wasn't aware of this guideline, but just saw it on the wiki. Please be aware that [this post](https://forum.kodi.tv/showthread.php?tid=353818) implies we can use other log levels, it may be helpful to update it with this information.

Reply:
cheers, i've added a note to the post as you've suggested.